app/src/test/java/org/simple/clinic/allpatientsinfacility/AllPatientsInFacilityLogicTest.kt

```diff
@@ -20,8 +20,8 @@ class AllPatientsInFacilityLogicTest {
   private val modelUpdates = PublishSubject.create<AllPatientsInFacilityModel>()
   private val testObserver = modelUpdates.test()
 
-  private val facility = PatientMocker.facility(UUID.fromString("1be5097b-1c9f-4f78-aa70-9b907f241669"))
-  private val user = PatientMocker.loggedInUser()
+  private val facility = TestData.facility(UUID.fromString("1be5097b-1c9f-4f78-aa70-9b907f241669"))
```

Comment:
It can, but the point of fixing the IDs is that if a test breaks for some reason, we can focus on what went wrong instead of trying to parse data which has also changed between consecutive test runs.

I don't see how having a different UUID here will make things easier to debug. Would you mind explaining where this is coming from?

Reply:
As we see the test names this is actually alright I guess.

Reply:
Discussed offline: It is unnecessary since we know which test has failed.